pkgs/development/python-modules/astropy/default.nix

```diff
@@ -123,11 +125,16 @@ buildPythonPackage rec {
   });
 
   nativeCheckInputs = [
+    hypothesis
     pytestCheckHook
     pytest-xdist
     pytest-astropy-header
-    pytest-astropy
+    pytest-doctestplus
+    pytest-remotedata
```

Comment:
How come the tests passed up until now without these deps?

https://cache.nixos.org/log/j2mdc9b36lvn79kcpmwn06jicxkcyfds-python3.12-astropy-7.1.0.drv

Reply:
because those are dependencies of pytest-astropy